eks_node_rollout/eks_node_rollout.py

```diff
@@ -227,9 +227,10 @@ def enable_autoscaling(asg_client, asg_name, dry_run=True):
 
 @click.command()
 @click.option('--cluster-name', envvar='EKS_NODE_ROLLOUT_CLUSTER_NAME', required=True, help="Cluster name to discover ASGs from")
+@click.option('--drain-timeout', envvar='EKS_NODE_ROLLOUT_DRAIN_TIMEOUT', default="120s", help="Timeout for draining worker node")
```

Comment:
Super minor nitpick, but can we get the syntax for the value somewhere in the help? Even if it's just a note on it being the same as the `kubectl` syntax for `--timeout`?